src/value-converters/README.md

```diff
@@ -1,8 +1,8 @@
 # Value Converters
 
-## anti-xss-html-sanitizer
+## diy-html-sanitiser
```

Comment:
Crummy name, but could not well keep the old one, once it's no longer the one and only. Better candidates?